checkout-operator.mk

```diff
@@ -7,9 +7,27 @@ define checkout_operator
 		git clone -b master https://github.com/$(1)/$(2).git $(SOURCE_DIR)/$(2) ;\
 	else \
 		if [[  -z "$${SKIP_GITREFRESH}" ]]; then \
-			pushd -n $(SOURCE_DIR)/$(2) && git reset --hard && git pull --force && popd -n ;\
+			pushd $(SOURCE_DIR)/$(2) && git reset --hard && git pull --force && popd ;\
```

Comment:
I notice the removal of `-n` - did it not work as intended?

Reply:
No, it doesn't change the stack.  So it's applying git reset/pull on the osd-operators-registry source.  Annoying to track down :)

Reply:
Doh 🤦‍♀️ I misread the manpage. The goal was to suppress output.

Reply:
@jewzaam Edited the first comment to do the suppression.

Reply:
Actually rather have the dir change logged.  There's a lot of noise already and it's helpful when looking at the output to see what happened with directory changes.